src/classes/app.py

```diff
@@ -142,8 +141,11 @@ def __init__(self, *args, mode=None):
         libopenshot_version = openshot.OPENSHOT_VERSION_FULL
         if mode != "unittest" and libopenshot_version < info.MINIMUM_LIBOPENSHOT_VERSION:
             QMessageBox.warning(
-                None, _("Wrong Version of libopenshot Detected"),
-                _("<b>Version %(minimum_version)s is required</b>, but %(current_version)s was detected. Please update libopenshot or download our latest installer.") % {
+                None,
```

Comment:
This will change our translation text...

Reply:
@jonoomph 
Tried to be careful about that, but I wasn't careful enough. Fixed so the translation master generates the same way.